packages/logs/README.md

```diff
@@ -667,6 +667,33 @@ window.DD_LOGS && DD_LOGS.logger.setHandler(['<HANDLER1>', '<HANDLER2>'])
 
 **Note**: The `window.DD_LOGS` check is used to prevent issues if a loading failure occurs with the SDK.
 
+### Access internal context
+
+After the Datadog browser logs SDK is initialized, you can access the internal context of the SDK. This will allow you to access the `session_id`.
+
+```
+getInternalContext (startTime?: 'number' | undefined)
+```
+You can optionally use `startTime` parameter to get the context of a specific time. If not return the current context is parameter is omitted.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏phrasing
```suggestion
You can optionally use `startTime` parameter to get the context of a specific time. If the parameter is omitted, the current context is returned.
```